src/initiatives/process.md

```diff
@@ -0,0 +1,57 @@
+# Process
+
+This page describes how lang team initiatives work. This is the process to use if you have an idea for a change you would like to make in the language.
+
+## Summary
+
+In a nutshell, the process for a successful initiative is as follows:
+
+- Have an idea
+  - Talk about it on internals, Zulip, etc to flesh it out a bit
+  - Ideally, identify a potential [owner]
+- Open a [proposal] as an issue on the lang-team repository
+  - A lang team member can decide to be your [liaison] and [_second_ your proposal][2nd].
+  - Once that happens, we will create a Zulip stream, tracking issue, and (optionally) repository, etc.
+- If warranted, [explore][experimental] the design space and author the RFC
+  - In this phase, the [owner] works with the [liaison] and other contributors to expore the design space and develop the RFC
+  - Code can be landed in this phase, but the feature gate is marked as "experimental"
+  - Users of the feature gate will get a warning that the RFC is under development
+  - Once the RFC is ready, it can be opened on the RFC repository and approved by the lang team
+- Finish [development]
+  - At this point, development proceeds but the feature gate does not have to be marked as "experimental"
+- [Feature complete]
+  - When the liaison feels that it is ready, the initiative may be declared "feature complete".
+  - This is primarily a 'signaling' mechanism to encourage testing and feedback.
```

Comment:
```suggestion
  - This is primarily a 'signaling' mechanism to the broader community to encourage testing and feedback.
```

I think it's worth emphasizing this is not intended for the lang team so much as the community, I think, right? The language team should be providing feedback on a roughly monthly cadence, though certainly the feature complete phase may signal a stronger desire to provide feedback alongside the community, as the feature is hopefully going to crystallize and not change too much.